docker/push.sh

```diff
@@ -19,7 +19,7 @@ TAG=${2:-beta}
 
 echo "Building AcousticBrainz web image with env $ENV tag $TAG..."
 docker build -t metabrainz/acousticbrainz:$TAG \
-        -f Dockerfile.prod \
+        --target acousticbrainz-prod
```

Comment:
We could use the `$ENV` here? but there's no `beta` target in the Dockerfile, so it would fail if we call this script with that as an argument

Reply:
I'm fine with it as is, to be honest.